diff --git a/security_settings.php b/security_settings.php
index 9e80c13..4c32fd3 100644
--- a/security_settings.php
+++ b/security_settings.php
@@ -613,6 +613,7 @@ print "<input type='hidden' name='redcap_csrf_token' value='".getCsrfToken()."'>
 	<td class="cc_data">
 		<select class="x-form-text x-form-field" style="padding-right:0; height:22px;" name="shibboleth_username_field">
 			<option value='none' <?php echo ($element_data['shibboleth_username_field'] == "none" ? "selected" : "") ?>><?php echo $lang['system_config_45'] ?></option>
+			<option value='HTTP_GLID' <?php echo ($element_data['shibboleth_username_field'] == "HTTP_GLID" ? "selected" : "") ?>>HTTP_GLID</option>
 			<option value='REMOTE_USER' <?php echo ($element_data['shibboleth_username_field'] == "REMOTE_USER" ? "selected" : "") ?>>REMOTE_USER</option>
 			<option value='HTTP_REMOTE_USER' <?php echo ($element_data['shibboleth_username_field'] == "HTTP_REMOTE_USER" ? "selected" : "") ?>>HTTP_REMOTE_USER</option>
 			<option value='HTTP_AUTH_USER' <?php echo ($element_data['shibboleth_username_field'] == "HTTP_AUTH_USER" ? "selected" : "") ?>>HTTP_AUTH_USER</option>
